pypeit/reduce.py

```diff
@@ -586,7 +595,9 @@ def find_objects(self, image, std_trace=None,
             skymask = skymask_pos
 
         if show:
-            self.show('image', image=image*(self.sciImg.fullmask == 0), chname='objfind',sobjs=sobjs_obj_single, slits=True)
+            indx = self.sciImg.boolean_mask(invert=True).astype(float)
```

Comment:
Let's call this gpm or something, not indx. Indx is not a good variable name for a mask.

Reply:
Done.